packages/serializers/html/src/serializeHtml.ts

```diff
@@ -71,21 +83,18 @@ export const serializeHtml = <V extends Value>(
       return elementToHtml<V>(editor, {
         props: {
           element: node as EElement<V>,
-          children: encodeURIComponent(
-            serializeHtml(editor, {
-              nodes: node.children as EDescendant<V>[],
-              preserveClassNames,
-              stripWhitespace,
```

Comment:
I was able to get the serialized html with proper <br /> by passing the `convertNewLinesToHtmlBr` here as well.